coordinator-common/src/main/java/org/apache/kafka/coordinator/common/runtime/CoordinatorLoader.java

```diff
@@ -49,51 +49,7 @@ public short unknownType() {
      * Object that is returned as part of the future from load(). Holds the partition load time and the
      * end time.
      */
-    class LoadSummary {
-        private final long startTimeMs;
-        private final long endTimeMs;
-        private final long schedulerQueueTimeMs;
-        private final long numRecords;
-        private final long numBytes;
-
-        public LoadSummary(long startTimeMs, long endTimeMs, long schedulerQueueTimeMs, long numRecords, long numBytes) {
-            this.startTimeMs = startTimeMs;
-            this.endTimeMs = endTimeMs;
-            this.schedulerQueueTimeMs = schedulerQueueTimeMs;
-            this.numRecords = numRecords;
-            this.numBytes = numBytes;
-        }
-
-        public long startTimeMs() {
-            return startTimeMs;
-        }
-
-        public long endTimeMs() {
-            return endTimeMs;
-        }
-
-        public long schedulerQueueTimeMs() {
-            return schedulerQueueTimeMs;
-        }
-
-        public long numRecords() {
-            return numRecords;
-        }
-
-        public long numBytes() {
-            return numBytes;
-        }
-
-        @Override
-        public String toString() {
-            return "LoadSummary(" +
-                "startTimeMs=" + startTimeMs +
-                ", endTimeMs=" + endTimeMs +
-                ", schedulerQueueTimeMs=" + schedulerQueueTimeMs +
-                ", numRecords=" + numRecords +
-                ", numBytes=" + numBytes + ")";
-        }
```

Comment:
Record the default toString method will show "Square brackets" instead "Parentheses", ie. `LoadSummary(...)` will change to  `LoadSummary[...]`, If won't broken any test, I think it will be OK.